src/components/event-card/event-card.component.tsx

```diff
@@ -0,0 +1,51 @@
+import { ReactElement } from "react";
+import Image from "next/image";
+
+import HugeiconsCalendar03 from "@/icons/HugeiconsCalendar03";
+import HugeiconsLocation04 from "@/icons/HugeiconsLocation04";
+import HugeiconsShoppingBag02 from "@/icons/HugeiconsShoppingBag02";
+
+import { AvailableEvent } from "@/lib/data";
+
+import styles from "./event-card.module.css";
+
+type Props = {
+  eventsData: AvailableEvent;
```

Comment:
از اسامی ساده استفاده کنید که منظور رو میرسونه
از طرفی بهتره خیلی سمت کلمه‌ی Event نرید چون تو JavaScript معنای خاص خودشو داره
به جاش از Activity میتونید استفاده کنید

```suggestion
  activity: Activity;
```